RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -137,3 +137,84 @@ boolean auto_fightLocketMonster(monster mon)
 	return true;
 
 }
+
+void juneCleaverChoiceHandler(int choice)
+{
+	switch(choice) {
+		case 1467: // Poetic Justice
+			run_choice(3); // +5 adventures, get beaten up
```

Comment:
I think this would get wrongly counted into auto_beatenUpCount if not prevented?
autoscend also aborts if it's not able to remove the beaten up debuff because it reasons that it can't face the monster that just beat it with even lower stats, there is a situation #1145 since it follows a victory can bypass the abort and keep adventuring with the debuff safely enough though it seems like it would be more complicated to do so here and then also keep track of the 5 turns of it